doc/nrf/protocols/thread/overview/architectures.rst

```diff
@@ -28,6 +28,33 @@ This can be done per platform, while retaining the ability to default to a stand
 
    OpenThread architecture adapted from `openthread.io <OpenThread system architecture_>`_
 
+|NCS| provides two different architecture options for OpenThread.
+The first one uses Zephyr networking layer to communicate OpenThread stack with the IEEE 802.15.4 radio driver.
+The second one uses the OpenThread stack directly to communicate with the IEEE 802.15.4 radio driver.
+
+To learn more about the differences between the two architectures, see the following table:
+
+.. list-table:: Thread architecture options
+   :widths: auto
+   :header-rows: 1
+
+   * - Integration with Zephyr networking layer.
+     - Direct IEEE 802.15.4 radio integration with OpenThread stack.
+   * - Uses Zephyr networking layer to communicate OpenThread stack with the IEEE 802.15.4 radio driver.
+     - Uses the OpenThread stack directly to communicate with the IEEE 802.15.4 radio driver.
+   * - Integrates Zephyr networking features, such as Sockets, IPv6, DNS, DHCPv6, zperf, etc.
```

Comment:
Maybe it's worth to add additional point (not extending this one..) about - which application layers are available e.g. CoAP and MQTT from Zephyr are only available in the first option.